cli/inputflags/flags.go

```diff
@@ -28,6 +29,7 @@ func (f *Flags) Options() anyio.ReaderOpts {
 func (f *Flags) SetFlags(fs *flag.FlagSet, validate bool) {
 	fs.StringVar(&f.Format, "i", "auto", "format of input data [auto,zng,zst,json,ndjson,zeek,zjson,csv,parquet]")
 	fs.BoolVar(&f.ZNG.Validate, "validate", validate, "validate the input format when reading ZNG streams")
+	fs.IntVar(&f.ZNG.Threads, "threads", 0, "number of threads used for scanning ZNG input")
```

Comment:
Nits: I don't think this knob is worth its weight. But if we keep it around, I think the field should be `Goroutines` and the flag should be `zng.goroutines`.

Reply:
I would hope that most people who use zq don't care that it's implemented in Go and more people know what a thread is compared to a goroutine.

Reply:
"Threads" made me think of runtime.LockOSThread.

But I suppose no one will use this so the name doesn't really matter.